website/content/docs/operators/deployment/schedulers/kubernetes.md

```diff
@@ -1,189 +1,286 @@
 ---
 title: Kubernetes
+new: true
 ---
 
-Heron supports deployment on [Kubernetes (k8s)](https://kubernetes.io/). Deployments on Kubernetes 
-are done via Docker and the k8s API.
+Heron supports deployment on [Kubernetes](https://kubernetes.io/) (sometimes called **k8s**). Heron deployments on Kubernetes use Docker as the containerization format for Heron topologies and use the Kubernetes API for scheduling.
+
+You can use Heron on Kubernetes in multiple environments:
+
+* Locally using [Minikube](#minikube)
+* In the cloud on [Google Container Engine](#google-container-engine) (GKE)
+* In [any other](#general-kubernetes-clusters) Kubernetes cluster
+
+## Requirements
+
+In order to run Heron on Kubernetes, you will need:
+
+* A Kubernetes cluster with at least 3 nodes
```

Comment:
should we add or Minikube here?
A Kubernetes cluster with at least 3 nodes or Minikube